### DISCUSSION TITLE
トラックボールが反応しなくなった

### DISCUSSION THREAD

**Discussion by HideoFukuda369** [Yowkees/keyball#224] [Q&A]
表題の通りです。
全てのキー、ボール共に問題なく1ヶ月ほど使用しておりましたが本日突然ボールのみ反応しなくなりました。

現在もキーは全て正常に反応しております。

このような場合どのように対処すればよいかご教授いただけないでしょうか

追記
下記の内容は一通りチェックし、はんだ付けし直しも試しましたがだめでした。。
https://github.com/Yowkees/keyball/discussions/51#discussioncomment-1044412

電源が入っている状態でL字からセンサー部分を抜き差しするとPCからデバイスの取り付け、取り外し音が鳴るので、そこは反応があるようです

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

電源が入った状態でＬ字コンスルーからセンサー基板を抜き差しするのは危険です。センサーが破損する場合があります。

一度組みなおして、ファームウェアを書き直してみるとどうでしょうか？お手数ですがご確認お願いいたします。

**Reply by HideoFukuda369** [reply]
ファームウェア書き込み直し試しましたが駄目でした。

**Comment by Yowkees** [comment]
ご連絡が遅くなり大変申し訳ありませんでした。

はんだ付けもやり直していただけたということで、センサーの故障の可能性が濃厚かと思います。
部品単位での保守部品等、販売可能です。
発送先等やりとりしたいので、白銀ラボウェブサイトのお問合せから一度ご連絡いただけないでしょうか。
https://shirogane-lab.com/pages/contact
お手数ですがご対応お願いいたします。
